Lombiq.Tests.UI.Samples/Tests/BasicVisualTests.cs

```diff
@@ -0,0 +1,173 @@
+using Lombiq.HelpfulLibraries.Common.Utilities;
+using Lombiq.Tests.UI.Attributes;
+using Lombiq.Tests.UI.Constants;
+using Lombiq.Tests.UI.Extensions;
+using Lombiq.Tests.UI.Services;
+using OpenQA.Selenium;
+using Shouldly;
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Processing;
+using System.Globalization;
+using System.IO;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Lombiq.Tests.UI.Samples.Tests;
+
+// In this basic test we will check visually the rendered content.
+public class BasicVisualTests : UITestBase
+{
+    public BasicVisualTests(ITestOutputHelper testOutputHelper)
+        : base(testOutputHelper)
+    {
+    }
+
+    // Checking that everything is OK with the branding on navbar of homepage.
+    // For this magic we are using ImageSharp.Compare package. You can find more info about it here:
+    // https://github.com/Codeuctivity/ImageSharp.Compare/blob/1.2.11/README.md.
+    [Theory, Chrome]
+    public Task AnonymousHomePageShouldExist(Browser browser) =>
+        ExecuteTestAfterSetupAsync(
+            context =>
+            {
+                context.SetViewportSize(CommonDisplayResolutions.HdPlus);
+
+                var testDumpFolderName = nameof(AnonymousHomePageShouldExist);
+                var testTempRootFolder = FileSystemHelper.EnsureDirectoryExists(Path.Combine("Temp", testDumpFolderName));
+
+                // We take a screenshot and save it to a temporary folder, and append temporary file to failure dump.
+                var fullScreenImageFileName = "navbar_full_screen.bmp";
+                var fullScreenImageTempFileName = Path.Combine(testTempRootFolder, fullScreenImageFileName);
+                context.TakeScreenshot(fullScreenImageTempFileName);
+                context.AppendFailureDump(
+                    Path.Combine(testDumpFolderName, fullScreenImageFileName),
+                    context => Task.FromResult((Stream)File.OpenRead(fullScreenImageTempFileName)));
+
+                var navbarElementSelector = By.ClassName("navbar-brand");
+                // We set the browser's window size, dpi and scale settings explicitly to make the test environment
+                // similar on every platforms, but we have no control over the fonts. In this case, the selected block
+                // element size, mainly height is depends of the used font on a given platform.
+                // We don't know the exact size of the selected element, so we should select the interested region for
+                // reference and captured bitmaps, and use it at the end. Without cropping the reference and captured
+                // bitmap to same size, the ImageSharpCompare.CalcDiff(...) throws an ImageSharpCompareException.
+                var cropRegion = new Rectangle(0, 0, 1400, 23);
+
+                // First we take a screenshot of element area. This will be compared to a reference image what we
+                // prepared before.
```

Comment:
You can find such issues easily with Grammarly, if you go through your own PR and select such lines for code suggestions.

```suggestion
                // First, we take a screenshot of the element area. This will be compared to a reference image that we
                // prepared before.
```